### PR TITLE
create users table in dynamodb

### PR DESCRIPTION
**Pull request by ahsankamal**
Issue No: https://github.com/sunnah-com/corrections/issues/29

### CODE REVIEW

File: frontend/scripts/init-dynamodb.sh [hunk 1]
```diff
@@ -101,4 +107,20 @@ aws dynamodb put-item --table-name HadithCorrectionsArchive --item '{
    "moderatedBy":{
       "S":"rootuser"
    }
-}' --endpoint-url http://dynamodb-local:8000 --region us-west-2
\ No newline at end of file
+}' --endpoint-url http://dynamodb-local:8000 --region us-west-2
+
+aws dynamodb put-item --table-name Users --item '{
+    "username":{
+        "S": "ahsan"
```
Comment (hasankhan) [inline frontend/scripts/init-dynamodb.sh:114]: can we change this to 'guest' please?

File: frontend/scripts/init-dynamodb.sh [hunk 2]
```diff
@@ -101,4 +107,20 @@ aws dynamodb put-item --table-name HadithCorrectionsArchive --item '{
...
+    },
+    "permissions":{
+        "M":{
+              "manage users": {
+                "BOOL": true
+              },
+              "queues":{
+                "SS": ["global", "skip", "secondary"]
```
Comment (hasankhan) [inline frontend/scripts/init-dynamodb.sh:122]: let's remove 'skip', we decided yesterday to drop that queue

File: frontend/scripts/init-dynamodb.sh [hunk 3]
```diff
@@ -101,4 +107,20 @@ aws dynamodb put-item --table-name HadithCorrectionsArchive --item '{
...
\ No newline at end of file
+}' --endpoint-url http://dynamodb-local:8000 --region us-west-2
+
+aws dynamodb put-item --table-name Users --item '{
+    "username":{
+        "S": "ahsan"
+    },
+    "permissions":{
+        "M":{
+              "manage users": {
```
Comment (hasankhan) [inline frontend/scripts/init-dynamodb.sh:118]: can we replace the space with an underscore?

**Review by hasankhan — CHANGES_REQUESTED** [review]
JazakAllah khair added two comments

File: frontend/scripts/init-dynamodb.sh (hunk 1 above)
Comment (ahsankamal) [reply frontend/scripts/init-dynamodb.sh]: Done.

File: frontend/scripts/init-dynamodb.sh (hunk 2 above)
Comment (ahsankamal) [reply frontend/scripts/init-dynamodb.sh]: removed 'skip' from the set.

File: frontend/scripts/init-dynamodb.sh (hunk 3 above)
Comment (ahsankamal) [reply frontend/scripts/init-dynamodb.sh]: done

File: frontend/scripts/init-dynamodb.sh (hunk 2 above)
Comment (ahsankamal) [reply frontend/scripts/init-dynamodb.sh]: Please let me know if any other changes are required in this PR.

**Pull request merged**